logging_manager/src/StorageManager.py

```diff
@@ -0,0 +1,59 @@
+# Library imports
+import os
+import shutil
+import psutil
+import time
+
+def storage_manager_main():
+    storage_directory = "/media/files"
+
+    # Read data from tmp directory
+    tmp_path = "/tmp"
+
+    for files in os.listdir(tmp_path):
+        file_path = os.path.join(tmp_path, files)
+
+        if os.path.isfile(file_path) and files.endswith(".json"):
+            print(files)
+            with open(file_path, 'r') as file:
+                read_data = file.read()
+                print(read_data)
+
+        else:
+            print("File not found")
+
+    # Check if device is plugged
+    # If no device detected, wait 30 seconds then try again
+    while not os.path.exists(storage_directory):
```

Comment:
So if the storage directory does exist, then nothing inside this while will execute? Think the logic is inverted here, or maybe an if statement first, lets review tomorrow